lib/vericite.rb

```diff
@@ -263,7 +263,7 @@ def sendRequest(command, args)
           external_content_data.upload_content_length = args[:psize]
           report_meta_data.external_content_data = external_content_data
           # @return [Array<ExternalContentUploadInfo>]
-          data, status_code, headers = vericite_client.reports_submit_request_context_id_assignment_id_user_id_post(context_id, assignment_id, user_id, consumer, consumer_secret, report_meta_data)
+          data, status_code, headers = vericite_client.reports_submit_request_context_id_assignment_id_user_id_post(context_id, assignment_id, user_id, consumer, consumer_secret, report_meta_data, {:'encrypted' => true})
```

Comment:
I'll update and test the format. 

Yes, it is always encrypted. This is essentially a flag to indicate to our api to use the new call/response headers for encryption security. We are phasing out the old method and we use this flag to be able to accomidate the hundreds of LMS integrations updating at different times.  I could tuck this away within the vericite gem if you prefer.